app/actions/service_credential_binding_create.rb

```diff
@@ -0,0 +1,68 @@
+module VCAP::CloudController
+  module V3
+    class ServiceCredentialBindingCreate
+      class UnprocessableCreate < StandardError
+      end
+
+      def initialize(user_audit_info)
+        @user_audit_info = user_audit_info
+      end
+
+      def precursor(service_instance, app: nil, name: nil)
```

Comment:
I find this name not that descriptive of what it does. It just shows when it does it. 
It's really doing "validate and create binding" but that doesn't help. We might need to think about this later

Reply:
I agree, I don't like this name. I thought about calling it `create_binding` -- that's what it's actually doing. I chose to keep it consistent with the route bindings though. I guess we should change it to a name that works for both bindings?

Note that, as mentioned in another comment below, this particular method is NOT running validations.